tools/x-repo-types/Makefile

```diff
@@ -39,6 +39,13 @@ goal-v-sdk-stateproof:
 		--y-package "github.com/algorand/go-algorand-sdk/v2/types" \
 		--y-type "StateProof"
 
+goal-v-sdk-consensus:
```

Comment:
This change is purely for convenience. The Consensus xrt test case was [already defined](https://github.com/algorand/go-algorand/blob/c6f3769272c85734526b13c670e2bd844ad156d7/tools/x-repo-types/xrt_test.go#L76-L83).